src/app/sessions/containers/sessions-create/sessions-create.component.ts

```diff
@@ -43,11 +45,17 @@ export class SessionsCreateComponent implements OnInit {
     }
 
     create(session) {
+        this.sessionId = session.id;
         this.sessionCreationService.create(session);
-        this.dialogRef = this.openDialog(session);
+        this.dialogRef = this.openDialog();
+        this.dialogRef.afterClosed().subscribe(() => this.getCreatedSession(this.sessionId));
     }
 
-    private openDialog(session) {
+    public getCreatedSession(sessionId: string) {
```

Comment:
name here is confusing, get indicates that you will return sth, but you just dispatch an action